slices/liblmdb0.yaml

```diff
@@ -0,0 +1,15 @@
+package: liblmdb0
+
+essential:
+  - liblmdb0_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/liblmdb.so.0*:
+
+  copyright:
+    contents: 
```

Comment:
There appears to be a trailing space character at the end of this `contents:` line, can you remedy?